Assets/Scripts/PlayerInventory.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/**
+    Klasa ekwipunku dla gracza.
+    Umożliwia używanie przedmiotów.
+**/
+public class PlayerInventory : Inventory {
+
+    public float fadeInScale = 25f;
+    public float fadeOutScale = 15f;
+    public Sprite emptySlotSprite;
+
+    public PlayerControlls player;
+    public Canvas canvas;
+
+    UnityEngine.UI.Image[] images;
+    int lastSelected = 0;
+    ItemObject[] equipped;
+    ItemObject[] picked;
+
+	// Use this for initialization
+	void Start () {
+        images = new UnityEngine.UI.Image[8];
```

Comment:
Why is it done on Start() and not when declaring member? Also maybe we could extract the constant out.